hamocc/hamocc4bcm.F90

```diff
@@ -66,6 +66,7 @@ SUBROUTINE HAMOCC4BCM(kpie,kpje,kpke,kbnd,kplyear,kplmon,kplday,kldtday,&
 !  *REAL*    *dust*       - dust deposition flux [kg/m2/month].
 !  *REAL*    *rivin*      - riverine input [kmol m-2 yr-2].
 !  *REAL*    *ndep*       - nitrogen deposition [kmol m-2 yr-2].
+!  *REAL*    *oaflx*      - alkalinity flux from alkalinization [kmol m-2 yr-2]
```

Comment:
Maybe I miss something, but typos in the rivin, ndep, oaflx units (yr-1 ?!)?

Reply:
Corrected